src/main/kotlin/com/exactpro/th2/common/module/provider/FileConfigurationProvider.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020-2022 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.common.module.provider
+
+import com.exactpro.th2.common.ConfigurationProvider
+import com.exactpro.th2.common.schema.configuration.Configuration
+import com.fasterxml.jackson.databind.ObjectMapper
+import java.nio.file.Paths
+
+
+class FileConfigurationProvider(
+    private val objectMapper: ObjectMapper,
```

Comment:
This argument looks strange, because implementation of ObjectMapper depends on file extension. ConfigurationProvider should decide what extension and what implementation should be used instead of an external entity.